src/server/server_request_responder.hpp

```diff
@@ -163,7 +163,8 @@ template<typename T_CONN>
 server_request_responder<T_CONN>::server_request_responder ()
   : m_threads_context_manager (std::make_unique<cubthread::system_worker_entry_manager> (TT_SYSTEM_WORKER))
 {
-  const auto THREAD_COUNT = std::thread::hardware_concurrency ();
+  const auto THREAD_COUNT = 4; // std::thread::hardware_concurrency ();
```

Comment:
Without this change, It caused failure to start server on my machine when I use two of server_request_responder. One is the existing one from PS to TS, and another is what I've added for PS->PS. However, without adding the new one, it would cause the failure in a many-core machine. Please look into the issue I created : http://jira.cubrid.org/browse/LETS-788.

Welcome to opinions on this.